packages/docs/developer-resources/integrations/listings.md

```diff
@@ -0,0 +1,123 @@
+
+The Celo Protocol Github is located [here.](https://github.com/celo-org/)
+
+
+## Outline
+In this guide you will:
+* Go through an audit checklist to ensure top security requirements are met prior to running your node.
+* Run a Celo blockchain network node.
+* Find the relevant brand assets for the Celo protocol.
+* How to integrate Celo protocol into your infrastructure.
+* Learn about the name change from cGLD to CELO.
+* Learn about useful API endpoints for your listing needs.
+* Go over the NPM documentation for your listing needs.
+* Learn about the Stability protocol and whitepaper.
+
+## Audits
+All the security audits on the smart contracts, security and economics of the Celo Platform can be found [here](https://celo.org/audits).
+
+## Running a Celo Node
+In order to get started with the Celo Platform for your listing needs, you need to be able to run a Celo full-node. This helps you audit the distributed ledger by having your own node running. The documentation on running a full-node are found [here](https://docs.celo.org/getting-started/mainnet/running-a-full-node-in-mainnet).
+
+## Celo Brand Assets for Listing
+If you are listing Celo on your exchange, you will probably need access to the Celo Platform brand assets. They can be found [here](https://celo.org/experience/brand#overview).  
+
+Please ensure your use of the Celo Platform assets provided follows our brand policy found [here](https://celo.org/brand-policy).
+
+## Integrating Celo With Your Infrastructure
+There are several ways to integrate the Celo Platform with your infrastructure.
+
+A general overview of integrations that would be relevant to you listing Celo Platform are shown [here](https://docs.celo.org/developer-guide/overview/integrations/general).
+
+For more specific use-cases for exchanges, please checkout the [Custody Integration Guide](https://docs.celo.org/developer-guide/overview/integrations/custody) as well.
+
+
+## Celo Native Asset and Celo Dollar
+
+There are two important assets on the Celo network, the Celo native asset (CELO) and the Celo Dollar (cUSD). CELO was called Celo Gold (cGLD) when the contract was deployed, so you will often see references to Celo Gold in the codebase. CELO and cGLD are the same thing. You can [view the CELO implementation here](https://explorer.celo.org/address/0x8dd4f800851db9dc219fdfaeb82f8d69e2b13582/contracts)
+
+CELO has an ERC20 interface, so users can interact with CELO via the token standard, but it is important to note that not all CELO transfers are required to go through the token contract. CELO can also be transferred by specifying the value field of a transaction, in the same way that ETH can be transferred in Ethereum. To properly monitor balance changing operations of CELO, it can be helpful to use  [Celo Rosetta.](https://github.com/celo-org/rosetta).  Celo Rosetta provides an easy way to obtain changes that are not easily queryable using the celo-blockchain RPC.
```

Comment:
```suggestion
CELO has an ERC20 interface, so users can interact with CELO via the token standard, but it is important to note that not all CELO transfers are required to go through the token contract. CELO can also be transferred by specifying the value field of a transaction, in the same way that ETH can be transferred in Ethereum. To properly monitor balance changing operations of CELO, it can be helpful to use [Celo Rosetta.](https://github.com/celo-org/rosetta). Celo Rosetta provides an easy way to obtain changes that are not easily queryable using the celo-blockchain RPC.
```